app/controllers/users.js

```diff
@@ -19,3 +20,20 @@ exports.signUp = async (req, res, next) => {
     return next(error);
   }
 };
+
+exports.signIn = (req, res, next) => {
+  try {
+    const { user } = res.locals;
+
+    const payload = {
+      id: user.id,
+      email: user.email
+    };
+    const secret = 'hola1234';
+    const token = jwt.encode(payload, secret);
```

Comment:
This logic shouldnt be here. You could have an service called sessions_manager.js. This file contains the logic to:

- Encrypt
- Decrypt
- Generate Access Token
- Login

And in controller, only calls these functions